md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/UserRolesAPI.java

```diff
@@ -83,24 +80,21 @@ RestWrapper list(
      * This method calls the proc deleteUserRoles and deletes a record corresponding to the id passed.
      *
      * @param userRoleId
-     * @param model
      * @return nothing.
      */
     @RequestMapping(value = "/{urid}", method = RequestMethod.DELETE)
-    public
+
     @ResponseBody
-    RestWrapper delete(
-            @PathVariable("urid") Integer userRoleId, Principal principal,
-            ModelMap model) {
+    public RestWrapper delete(
+            @PathVariable("urid") Integer userRoleId, Principal principal) {
         RestWrapper restWrapper = null;
         try {
             userRolesDAO.delete(userRoleId);
-            // s.delete("call_procedures.DeleteUserRoles", userroles);
-
             restWrapper = new RestWrapper(null, RestWrapper.OK);
             LOGGER.info("Record with ID:" + userRoleId + " deleted from UserRoles by User:" + principal.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Record with ID:" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)